etl/steps/data/grapher/fasttrack/2022-02-28/nuclear_warhead_inventories.meta.yml

```diff
@@ -2,7 +2,7 @@ dataset:
   namespace: fasttrack
   version: '2022-02-28'
   short_name: nuclear_warhead_inventories
-  title: Nuclear warhead inventories – Federation of American Scientists
+  title: DRAFT Nuclear warhead inventories – Federation of American Scientists
```

Comment:
Do we want these changes to persist?

Reply:
Yes, we already have this dataset in grapher, so I'd rather un-DRAFT it after Bastian takes ownership of it. (Alternatively I might remove it once we have some real datasets there)